index.js

```diff
@@ -262,7 +264,9 @@ alexa.session = function(session) {
   }
   this.getAttributes = function() {
     // do some stuff with session data
```

Comment:
This feels like a useless comment that was left in there, feel free to better document these methods. Thanks!